kaggle/api/kaggle_api_extended.py

```diff
@@ -2262,22 +2267,6 @@ def kernels_list(self,
       request.search = search or ''
       return kaggle.kernels.kernels_api_client.list_kernels(request).kernels
 
-    kernels_list_result = self.process_response(

```

Comment:
Was this meant to be removed?

Reply:
Yes.

Reply:
Yes, because that was dead code. The `return` on the line above is the end of the method. You're right, I should have made a separate PR for this. But it seemed trivial, so I didn't.